app/build/.gitkeep

```diff
@@ -0,0 +1 @@
+# Keep directory in git.
```

Comment:
TIL about .gitkeep 🎉

Reply:
Not really a feature but more of a convention:

> A common, standardised practice to solve this exact issue, however, is to push a file called .gitkeep into your empty folders.

> This isn't a feature of Git! So you could name it anything. There's nothing special about the name .gitkeep – some developers add .gitignore instead, for example.

https://www.freecodecamp.org/news/what-is-gitkeep/